src/package.xml

```diff
@@ -847,6 +847,7 @@
         <members>Batch__c.Object_Name__c</members>
         <members>Contact.Address_Verification_Status__c</members>
         <members>Contact.Batch__c</members>
+        <members>Contact.CountGiftsLastYear__c</members>
```

Comment:
@meena-sfdc A new field for testing purposes should not be added. If you need a test that does a count to a field, you can select any other numeric field on the Contact. Thanks.

Reply:
@force2b , I cannot find a field which is not referenced in the test, let me check again.

Reply:
Thanks. There are quite a lot of numeric fields on the Contact object between all the various rollups, but only some of them are used in the tests. I'm pretty sure (but not 100% positive) that there is a free Numeric field you can use for this.